File: VB/11_Images/RemoveImages.vb
Imports System.ComponentModel
Imports System.Text
Imports Spire.Presentation
Imports Spire.Presentation.Collections
Imports Spire.Presentation.Drawing.Animation

Namespace RemoveImages
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub btnRun_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnRun.Click
			'Create PPT document and load file
			Dim presentation As New Presentation()
			presentation.LoadFromFile("..\..\..\..\..\..\Data\RemoveImages.pptx")
			'Get the first slide
			Dim slide As ISlide = presentation.Slides(0)

			For i As Integer = slide.Shapes.Count-1 To 0 Step -1
				'It is the SlidePicture object
				If TypeOf slide.Shapes(i) Is SlidePicture Then
					slide.Shapes.RemoveAt(i)
				End If
			Next i

			Dim result As String = "RemoveImages_result.pptx"

			'Save to file.
			presentation.SaveToFile(result, FileFormat.Pptx2013)

			'Launch the PowerPoint file.
			PptDocumentViewer(result)
		End Sub

		Private Sub PptDocumentViewer(ByVal fileName As String)
			Try
				Process.Start(fileName)
			Catch
			End Try
		End Sub
	End Class
End Namespace

File: CS/11_Images/RemoveImages.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using Spire.Presentation;
using Spire.Presentation.Collections;
using Spire.Presentation.Drawing.Animation;

namespace RemoveImages
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();            
        }

        private void btnRun_Click(object sender, EventArgs e)
        {
            //Create PPT document and load file
            Presentation presentation = new Presentation();
            presentation.LoadFromFile(@"..\..\..\..\..\..\Data\RemoveImages.pptx");
            //Get the first slide
            ISlide slide = presentation.Slides[0];
      
            for (int i = slide.Shapes.Count-1; i >=0; i--)
            {
                //It is the SlidePicture object
                if (slide.Shapes[i] is SlidePicture)
                {
                    slide.Shapes.RemoveAt(i);
                }            
            }

            String result = "RemoveImages_result.pptx";

            //Save to file.
            presentation.SaveToFile(result, FileFormat.Pptx2013);

            //Launch the PowerPoint file.
            PptDocumentViewer(result);
        }

        private void PptDocumentViewer(string fileName)
        {
            try
            {
                System.Diagnostics.Process.Start(fileName);
            }
            catch { }
        }
    }
}